src/app/containers/Figure/index.test.jsx

```diff
@@ -36,6 +36,27 @@ describe('Figure', () => {
     expect(Object.keys(wrapper.props()).length).toBe(7);
   });
 
+  it('should load lazyload component and transition component when lazyLoad prop is set to true', () => {
+    const wrapper = mount(FigureLazyLoadImage).find(LazyLoad);
+    const { children: transitionComponent } = wrapper.props();
+    const {
+      timeout,
+      onExited,
+      onExiting,
+      onEntering,
+      onEntered,
+      children: imageWrapper,
+    } = transitionComponent.props;
+
+    expect(transitionComponent).toBeDefined();
+    expect(imageWrapper).toBeDefined();
+    expect(onExited).toBeDefined();
```

Comment:
Do the values of these not matter?

Reply:
@oluoluoxenfree those are handlers for the different states of the [transition component](http://reactcommunity.org/react-transition-group/transition), we're not actually using any one of them currently, just checking if they're defined to make sure it's the transition component